fixpatches/build.gradle.kts

```diff
@@ -16,10 +16,14 @@ kotlin {
                 implementation(libs.sarif4k)
```

Comment:
@nulls it works fine in tests without including of `slf4j-api` and `log4j-api` do we need to include it anyway for transitive dependencies?

Reply:
no, we don't need: https://github.com/oshai/kotlin-logging/blob/master/build.gradle.kts
`mu-kotlin` contains all required apis. we contain\use `mu-kotlin` as api (slf4j-api is implementation for JVM only)